fix: tracker new vs deprecated docs links

teleivo opened the pull request:
and be more specific by always adding 'deprecated'/'new' in front of endpoints

teleivo committed:
fix: tracker new vs deprecated docs links

and be more specific by always adding 'deprecated'/'new' in front of
endpoints

enricocolasante commented on src/developer/web-api/tracker-old.md:
```diff
@@ -9,12 +9,12 @@
->  further assistance. NOTE: The feature for data sync(importMode=SYNC) is not implemented in the
->  new tracker endpoints, and if you are using this feature you will have to postpone the migration
->  until a new SYNC feature is in place.
+>* If you already use the deprecated tracker endpoints in production, please plan to migrate over to
```
```suggestion
>* If you are still using the deprecated tracker endpoints in production, please plan to migrate over to
```

enricocolasante commented on src/developer/web-api/tracker.md:
```diff
@@ -10,22 +10,21 @@
->* If you already use the tracker endpoints in production, please plan to migrate over to the new
->  version. [Migrating to new tracker
->  endpoints](https://docs.dhis2.org/en/develop/using-the-api/dhis-core-version-master/tracker-old.html#webapi_tracker_migration)
+>* If you already use the deprecated tracker endpoints in production, please plan to migrate over to
```
```suggestion
>* If you are still using the deprecated tracker endpoints in production, please plan to migrate over to
```

teleivo committed:
chore: write new tracker docs without deprecated endpoints

in mind. Remove any mention that these endpoints are not stable/production ready
